src/app/debug.tsx

```diff
@@ -165,8 +165,8 @@ const Debug = ( c: RenderContext ) => {
 				<figure className="promises">
 					{ promiseRejections.size ? (
 						<ul>
-							{ Array.from( promiseRejections.values() ).map( ( [ ts, reason ] ) => (
-								<li>
+							{ Array.from( promiseRejections.values() ).map( ( [ ts, reason ], i ) => (
+								<li key={ i }>
```

Comment:
yes this is silly but we don't really have a good key. the timestamp isn't unique and neither is the combo of ts and the reason. index will have to do.

Reply:
this seems fair to me. this list should both be small _and_ we should have no reason to believe we'll be reordering the items. since we can't tell React "don't worry about it" the `index` is a fine tradeoff - as would ignoring the warning